.github/workflows/cross-version-tests.yml

```diff
@@ -60,8 +60,26 @@ jobs:
         run: |
           conda install python=3.6
           python --version
+      - name: Get cache key
+        env:
+          INSTALL_COMMAND: ${{ matrix.install }}
+        id: get-cache-key
+        run: |
+          date=$(date -u "+%Y%m%d")
+          hash=$(echo -n "$INSTALL_COMMAND" | sha256sum)
+          echo "::set-output name=key::$ImageOS-$ImageVersion-wheels-$date-$hash"
+      - uses: actions/cache@v2
+        if: ${{ matrix.package == 'pyspark' && matrix.version == 'dev' }}
+        with:
+          path: /home/runner/.cache/wheels
+          key: ${{ steps.get-cache-key.outputs.key }}
```

Comment:
Cache key looks like this:

`ubuntu20-20210419.1-wheels-20210427-9e9891b7e99e56166e9a6e511cc7a92eb80a37e6a588b0b3a5233c6881e8c32d`

Reply:
GitHub update ubuntu images weekly:

![Screen Shot 2021-04-29 at 10 30 14](https://user-images.githubusercontent.com/17039389/116491915-f2351e80-a8d5-11eb-899f-2340f372b96c.png)